client/src/components/auth/RegisterPatient.component.js

```diff
@@ -0,0 +1,130 @@
+import React, { Fragment, useState } from 'react';
+import { connect } from 'react-redux';
+import { register } from '../../actions/auth';
+import { Link, Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+export const RegisterPatient = ({ register, isAuthenticated }) => {
+  const [formData, setFormData] = useState({
+    name: '',
+    accountType: 'patient',
```

Comment:
Do we need separate patient and provider register? Why not one user register?

Reply:
We do need to have separate registers, but I have yet to implement the logic to use our account type field.  I will work on this after our midterms.

Reply:
understood, please be aware that in the database mock-up-data for users, identifying string's key name is "role". if it is not a good name, let me know I can change it.

Reply:
We will need to tide it up to another verification, to make sure registering users are current patients and doctors.